### ISSUE TITLE
"Mythos" decks

### ISSUE THREAD

**Issue by Phrancis** [Cardshifter/Cardshifter#231]
So far I have put together what will be 6 mythologies (will likely add more in the future). We need to come up with 10-12 (or more) cards for each. Here they are, please edit your name in if/when you want to work on that one. I would like if each mythology was done by a different person, if possible, for the sake of variety. Here they are:

- Greek: Jay
- Norse: tentatively @Zomis
- Egyptian: @MGS1GreyFox 
- Chinese: @Phrancis
- Roman
- Hindu

Short guidelines:

- Refer to issue #223 for the upcoming rules/features of this mod
- Assume a maximum of 100 Mana per player, and that the used Mana will not be available while the card using it is in play
- For cards with high mana cost (e.g. 50+) consider making another card that allows you to pull it back into your hand or into the graveyard
- Stick to these 3 kinds of cards: Persons, Places, objects. Persons and Places should be playable creatures. Places should not be able to attack. Objects can be either creatures or enhancements, at your option. No spells. 
- You can commit directly to the `js` branch as long as each person is working on a different file. 

Any questions feel free to ask!

**Comment by Phrancis** [comment]
Here are a few useful links:

- [Cards](https://github.com/Cardshifter/Cardshifter/tree/js/extra-resources/mods/mythos)
- [Documentation](https://github.com/Cardshifter/Cardshifter/tree/js/Documentation)

**Comment by Zomis** [comment]
Note that the documentation does not match the actual format, as the format has changed recently from JavaScript to Groovy.

The format you're using doesn't matter much IMO, you could write the values of the cards in pure english if you would like - the groovy format isn't far from plain English anyway ;)

**Comment by Phrancis** [comment]
Right. I'll update the documentation once the DSL syntax is settled :)

**Comment by Phrancis** [comment]
FYI - [Documentation has been updated](https://github.com/Cardshifter/Cardshifter/tree/js/Documentation).

Also if you like some examples of cards to use as models, check out the [Chinese](https://github.com/Cardshifter/Cardshifter/blob/js/extra-resources/mods/mythos/chinese.groovy) deck.

**Comment by Phrancis** [comment]
To clarify one thing, I don't expect others to document each card the way I did for the Chinese, if you just come up with the card attributes and effects, I'll take care of the rest later. It'll be a while yet before we support images and such so it's no hurry.

**Comment by Phrancis** [comment]
Created `common.groovy` and added some cards. Anyone feel free to add more if you think of any.

**Comment by Phrancis** [comment]
Added QPaysTaxes | @newbiedoodle for Roman mythos

**Comment by Phrancis** [comment]
In case anyone needs inspiration/ideas: **[GodChecker.com](http://www.godchecker.com/)**
